doc/nrf/libraries/bluetooth_services/services/esl_client.rst

```diff
@@ -0,0 +1,88 @@
+.. _esl_service_client_readme:
+
+Electronic Shelf Label Service (ESLS) Client
+############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This module implements the Electronic Shelf Label Service client which acts as Access Point(AP) to use the ESL Service exposed by an ESL device,
+and how the central AP communicates with ESLs connectionlessly when one or more ESLs are in the Synchronized state.
+
+The ESL Service Client is used in the :ref:`central_esl` sample.
+
+Service UUID
+************
+
+The 128-bit vendor-specific service UUID is ``0x1857``.
+
+Characteristics
+***************
+
+This service has 9 characteristics.
+
+ESL Address Characteristic (``0x2BF6``)
+============================================================
```

Comment:
Same thing with the alignment as in the previous doc.
And actually, same changes as in the previous one, as they are almost identical.